packages/react/src/components/Autofill/Autofill.tsx

```diff
@@ -248,6 +246,8 @@ export class Autofill extends React.Component<IAutofillProps, IAutofillState> im
             if (this.props.enableAutofillOnKeyPress!.indexOf(ev.which) !== -1) {
               this._autoFillEnabled = true;
             }
+          } else if (this.cursorLocation && this.cursorLocation < this.state.inputValue.length) {
```

Comment:
Did you figure out why this only started to repro in v8? It doesn't seem like this particular code should have changed (though I haven't looked at the history), so it's a little surprising that the fix is here. Might be fine, just want to make sure we understand the root cause.

Reply:
I have been investigating this issue further and updated the pull request with some new modifications. Here is a list of the bugs I found from V8 and fixes:

**Bug 1:**
`setSelectionRange` in `componetDidUpdate` never is called. This causes the visible cursor to jump mentioned in the issue.

Current Fix:  Remove ```!autoFillEnabled``` from the else if within `componetDidUpdate`. However, this reveals another bug.

**Bug 2:**
`Autofill` runs controlled and uncontrolled scripts at the same time.

With the previous fix, another visual bug becomes noticeable. When typing, the cursor will jump back a single space upon reaching the end. This has to do with the input value updating in both **Autofill** and **ComboBox** through `getDerivedStateFromProps` and then `updateValue`. This behavior also applies to the `value` prop in **Autofill**.

Current Fix: `inputValue` should only update if props.value and deprecated props.updateValueInWillRecieveProps are not provided.

**Update To ComboBox:**
ComboBox's Autofill still uses deprecated updateValueInWillRecieveProps and defaultValue. It was updated to use value.

Reply:
![Autofill Bug 2 Example](https://user-images.githubusercontent.com/7192338/124210450-4b6d2a00-daa0-11eb-807f-f4125f8ae720.gif)

Here's a gif showing the multiple state calls and other cursor jump

Reply:
This is awesome, thanks for the detailed investigation!